roku_dev_cli/roku_dev_cli.py

```diff
@@ -572,6 +578,7 @@ def main():
     parser.add_argument('-a', '--automation', action='store_true', help="creates an automation build")
     parser.add_argument('-p', '--proxy', action='store_true', help="builds the app to proxy through this host")
     parser.add_argument('-s', '--proxy_scripts', nargs='+', help='Provide paths to proxy add-on scripts', required=False)
+    parser.add_argument('--proxy_exclude', nargs='+', help='Provide domains that will be exclude by proxy filter', default=[], required=False)
```

Comment:
👏👏👏👏👏👏👏

Very nice! Looks like this even supports RegExs as well. Think there’s any value in calling that out in `—help`?